gateway/src/service.rs

```diff
@@ -437,7 +437,7 @@ impl GatewayService {
         project_name: ProjectName,
         account_name: AccountName,
         is_admin: bool,
-        limit: Option<u32>,
+        can_create_project: bool,
         idle_minutes: u64,
     ) -> Result<FindProjectPayload, Error> {
```

Comment:
Technically this function should not take in an argument to check if a project can be created. I think it is fine for now since I  don't see an easy way to move the logic to the caller.